go-controller/pkg/ovn/ovn.go

```diff
@@ -754,6 +752,33 @@ func (oc *Controller) removeServiceEndpoints(lb, vip string) {
 	}
 }
 
+func (oc *Controller) assignPodAddresses(nodeName string) (net.HardwareAddr, *net.IPNet, error) {
+	var (
+		podMac    net.HardwareAddr
+		podIP     *net.IP
+		podIPMask *net.IPMask
+		err       error
+	)
+
+	// First try to get the pod addresses quickly then fall back to polling every second.
+	podIP, podIPMask, err = oc.lsCache.AllocateNextIP(nodeName)
+	if err != nil {
+		return nil, nil, err
+	}
+	podMac, err = hwaddr.GenerateHardwareAddr4(*podIP, hwaddr.PrivateMACPrefix)
+	if err != nil {
+		defer func() {
+			err := oc.lsCache.ReleaseIP(nodeName, podIP)
+			if err != nil {
+				klog.Error(err)
+			}
+		}()
+		return nil, nil, err
+	}
+	return podMac, &net.IPNet{IP: *podIP,
```

Comment:
As above, put each member of a multi-member struct creation on separate lines.

```
return podMac, &net.IPNet{
    IP: *podIP,
    Mask: *podIPMask,
}, nil
```

Reply:
ack